src/java.base/macosx/classes/sun/nio/fs/BsdNativeDispatcher.java

```diff
@@ -64,6 +64,29 @@ static byte[] getmntonname(UnixPath path) throws UnixException {
     }
     static native byte[] getmntonname0(long pathAddress) throws UnixException;
 
+    /**
+     * int clonefile(const char * src, const char * dst, int flags);
+     */
+    static int clonefile(UnixPath src, UnixPath dst, int flags)
+        throws UnixException {
+        if (src.getFileSystem() == dst.getFileSystem()) {
```

Comment:
Why does this check if getFileSystem() returns the same file system? There is only one instance of UnixFileSystem so this will always be true.

Will the "throws UnixException {" fit on the previous line to fixing the odd formatting.

Reply:
Fixed in 2157a1480229fa697c824ae814f11a309627d768.